gtest/src/system.rs

```diff
@@ -97,4 +98,12 @@ impl System {
         let code = fs::read(&path).unwrap_or_else(|_| panic!("Failed to read file {:?}", path));
         self.0.borrow_mut().store_new_code(&code)
     }
+
+    pub fn get_mailbox<ID: Into<ProgramIdWrapper>>(&'_ self, id: ID) -> Mailbox<'_> {
```

Comment:
Lifetime elision: https://doc.rust-lang.org/nomicon/lifetime-elision.html

```suggestion
    pub fn get_mailbox<ID: Into<ProgramIdWrapper>>(&self, id: ID) -> Mailbox {
```

Reply:
Seems it works without placeholded lifetimes. Should be remove for `get_program` also if it so